package.json

```diff
@@ -79,7 +79,7 @@
     "knockout": "3.4.0",
     "konva": "1.6.8",
     "magic-wand-js": "1.0.0",
-    "moment": "2.22.2",
+    "moment": "2.23.0",
```

Comment:
This also looks possible. Albeit testing will be more extensive than the above ckeditor update (there are 8 openemr scripts that include the moment.js script).

Reply:
Looking at their changelog, the changes appear very minimal: https://github.com/moment/moment/blob/develop/CHANGELOG.md#2230-see-full-changelog

Reply:
The moment stuff is testing well.